Overview.src.html

```diff
@@ -4034,11 +4074,14 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
     <p class="note no-backref"><code title>101</code> is included in
     <span>null body status</span> due to its use elsewhere. It does not affect this step.
 
-   <li><p>Let <var title>stream</var> and <var title>Content-Type</var> be the result of
-   <span title=concept-BodyInit-extract>extracting</span> <var title>body</var>.
+   <li><p>Let <var>bytes</var> and <var>Content-Type</var> be the result of
+   <span title=concept-BodyInit-extract>extracting</span> <var>body</var>.
 
-   <li><p>Set <var title>r</var>'s <span title=concept-Response-response>response</span>'s
-   <span title=concept-response-body>body</span> to <var title>stream</var>.
+   <li><p>Set <var>r</var>'s
+   <span title=concept-Response-readable-stream>readable stream</span> to the result of
+   <span title=concept-construct-fixed-ReadableStream>constructing</span> a fixed
+   <span title=concept-ReadableStream>ReadableStream</span> with a list whose single element is a
+   <code>Uint8Array</code> whose contents are <var>bytes</var>. Rethrow any exceptions.
```

Comment:
We should probably also say object after `Uint8Array`. Also, instead of "whose contents are" I think "whose buffer is" is more accurate. Do we want to spell out that the buffer is a new `ArrayBuffer`? I think we might. It seems you are already handling that throwing.